toolchain/semantics/semantics_node_kind.h

```diff
@@ -20,6 +20,8 @@ class SemanticsNodeKind : public CARBON_ENUM_BASE(SemanticsNodeKind) {
  public:
 #define CARBON_SEMANTICS_NODE_KIND(Name) CARBON_ENUM_CONSTANT_DECLARATION(Name)
 #include "toolchain/semantics/semantics_node_kind.def"
+
+  using EnumBase::Create;
```

Comment:
It's protected on EnumBase. The problem stems from wanting to use the raw enum for templates (since the class cannot be used, and I want to be able to include the kind as a template argument).

```
./toolchain/semantics/semantics_node.h:158:59: error: 'Create' is a protected member of 'Carbon::Internal::EnumBase<Carbon::SemanticsNodeKind, Carbon::Internal::SemanticsNodeKindRawEnum>'
      return SemanticsNode(parse_node, SemanticsNodeKind::Create(Kind), type_id,
                                                          ^
./common/enum_base.h:96:25: note: declared protected here
  static constexpr auto Create(RawEnumType value) -> EnumType {
```